app/assets/javascripts/components/application_agree_checkbox.js.coffee

```diff
@@ -0,0 +1,28 @@
+@ApplicationAgreeCheckbox= React.createClass
+  changeValue: (e) ->
+    if $(e.target).is(":checked")
+      @props.setApplicationField(@props.field, true)
```

Comment:
you can probably just do `@props.setApplicationField(@props.field, $(e.target).is(":checked")` as it returns either true or false